test/e2e_node/memory_manager_metrics_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/gstruct"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
+	"k8s.io/kubernetes/test/e2e/feature"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2emetrics "k8s.io/kubernetes/test/e2e/framework/metrics"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var _ = SIGDescribe("Memory Manager Metrics", framework.WithSerial(), feature.MemoryManager, func() {
```

Comment:
Should we combine these metrics into memory_manager_test.go?

Reply:
I actually followed the same convention as cpu manager:
https://github.com/kubernetes/kubernetes/blob/f2cfbf44b1fb482671aedbfff820ae2af256a389/test/e2e_node/cpu_manager_metrics_test.go

Reply:
I started the `_metrics_test.go` trend. I don't have strong feelings. Both approaches (separate files, conflate into everything) have pros and cons. I can help cleaning up if we decide so.

Reply:
We would test these jobs as part of an existing job then?

Reply:
I don't have a preference really.  Just want to make sure wherever we test Memory Manager, we also run the metrics test.  If that is done via just the normal serial job great!

Reply:
absolutely! regardless of on which file they sit, these tests should totally run the metrics test using the existing serial job. If this is not the case (IIRC we were fine but didn't check in a while) it's a problem I need to fix yesterday!

Reply:
I found that you all have a memory management presubmit.   I kicked it off and I notice that one of these tests failed.  It seems to be legit so please look into it before merging.

Reply:
Hey @kannon92 sorry for my ignorance but what do you mean by memory management presubmit?

Reply:
/test pull-kubernetes-node-kubelet-serial-memory-manager

This allows you to trigger the e2e tests for memory manager

Reply:
Oh I see. When I tried locally I didn't notice any issue, but here I did see the failure that you are referring to.
I pushed a fix couple of hours ago let's see if it helps

Reply:
Seems to be fine now 🙂